Sources/Turf/Geometries/LineString.swift

```diff
@@ -215,4 +215,132 @@ extension LineString {
         
         return closestCoordinate
     }
+
+    private func squareDistance(from: CLLocationCoordinate2D, to: CLLocationCoordinate2D) -> Double {
+        let dx = from.longitude - to.longitude
+        let dy = from.latitude - to.latitude
+        return dx * dx + dy * dy
+    }
+
+    public func simplified(radialTolerance: Double) -> LineString {
```

Comment:
This public method needs documentation. Or does it need to be public?

Reply:
Made it private